includes/class-woocommerce-civicrm-manager.php

```diff
@@ -606,29 +606,26 @@ public function add_contribution( $cid, &$order ) {
 
 		$payment_instrument = $this->map_payment_instrument( $order->get_payment_method() );
 		$source = $this->generate_source( $order );
-		$params = array(
+		$params = [
 			'contact_id' => $cid,
-			'total_amount' => $rounded_subtotal,
 			// Need to be set in admin page
-			'contribution_type_id' => $default_contribution_type_id,
+			'financial_type_id' => $default_contribution_type_id,
 			'payment_instrument_id' => $payment_instrument,
-			'non_deductible_amount' => number_format( 0, 2, $decimal_separator, $thousand_separator ),
-			'fee_amount' => number_format( 0, 2, $decimal_separator, $thousand_separator ),
 			'trxn_id' => $txn_id,
 			'invoice_id' => $invoice_id,
 			'source' => $source,
 			'receive_date' => $order_paid_date,
-			'contribution_status_id' => $contribution_status_id,
+			// 'contribution_status_id' => $contribution_status_id,
```

Comment:
like to know why this line is commented out!!.

Reply:
This probably actually **should** be hard-coded to 'Pending' and then a payment recorded for completed WooCommerce orders, as per what the Order API expects.